test/bin/encore.js

```diff
@@ -47,4 +47,45 @@ module.exports = Encore.getWebpackConfig();
             done();
         });
     });
+
+    it('Smoke test using webpack.config.babel.js', (done) => {
+        testSetup.emptyTmpDir();
+        const testDir = testSetup.createTestAppDir();
+
+        fs.writeFileSync(
+            path.join(testDir, '.babelrc'),
+            `
+{
+    "presets": [
+        ["es2015", { "modules": "commonjs"}]
+    ]
+}
+            `
+        );
+
+        fs.writeFileSync(
+            path.join(testDir, 'webpack.config.babel.js'),
+            `
+import Encore from '../../index.js';
+
+Encore
+    .setOutputPath('build/')
+    .setPublicPath('/build')
+    .addEntry('main', './js/no_require')
+;
+
+const config = Encore.getWebpackConfig();
+export default config;
+            `
+        );
+
+        const binPath = path.resolve(__dirname, '../', '../', 'bin', 'encore.js');
+        exec(`node ${binPath} dev --context=${testDir}`, { cwd: testDir }, (err, stdout, stderr) => {
+            if (err) {
+                throw new Error(`Error executing encore: ${err} ${stderr} ${stdout}`);
+            }
+
+            done();
+        });
+    });
```

Comment:
Could this test be inside `functional.js` instead? Or is there a reason to have it here?

Reply:
The issue comes from the way the `index.js` file is imported in `webpack.config.babel.js`, and tests in `functional.js` skip that step by [directly creating an instance of `WebpackConfig`](https://github.com/symfony/webpack-encore/blob/master/lib/test/setup.js#L52).

I guess we could move the whole test case in `functional.js` but it would still be some kind of smoke test.

Reply:
Nah, works for me here - I just wanted to make sure there was some reason. It's most important for me that it's tested :)